src/components/IconsGroup/IconsGroup.tsx

```diff
@@ -0,0 +1,32 @@
+import { GitHub, ImportContacts } from '@mui/icons-material';
+import { Grid, Link } from '@mui/material';
+
+export function IconsGroup() {
+  return (
+    <Grid
+      container
+      item
+      alignItems='center'
+      justifyContent='center'
+      mobile={3}
+      tablet={3}
+      columnGap={4}
+    >
+      <Link
+        target='_blank'
+        href='https://github.com/dorm-social-room-manager/dsrm-frontend/wiki'
+        color='inherit'
+      >
+        <ImportContacts> </ImportContacts>
+      </Link>
+
+      <Link
+        target='_blank'
+        href='https://github.com/dorm-social-room-manager'
+        color='inherit'
+      >
+        <GitHub></GitHub>
+      </Link>
```

Comment:
You should get icon-list with URLs as parameter and then use this to render icons. With this approach, where you are hard-coding links with icons, we won't be able to use in other components if we will need different icons

Reply:
Resolved in c2c7072